Versioner/PrinceOfVersions.swift

```diff
@@ -21,6 +21,14 @@ public class PrinceOfVersions: NSObject {
         public let result: Result<UpdateInfo, UpdateInfoError>
     }
 
+    /// The queue on which the completion handler is dispatched
+    ///
+    /// By default, OperationQueue of the URLSession's delegateQueue is set to default one (nil)
+    @objc public enum CallbackQueue: Int {
```

Comment:
Why not just use DispatchQueue, and let user set preferred one, similar to what Alamofire does?

Reply:
Changed to `DispatchQueue`